nvflare/private/fed/client/communicator.py

```diff
@@ -190,7 +190,7 @@ def pull_task(self, servers, project_name, token, ssid, fl_ctx: FLContext):
             size = len(task.payload)
             task.payload = fobs.loads(task.payload)
             task_name = task.payload.get_header(ServerCommandKey.TASK_NAME)
-            self.logger.info(
+            self.logger.debug(
```

Comment:
@yhwen can you comment on do we need this one?

Reply:
This needs to remain as info. This is an important message to show how large the data the client get from the server when fetching a task.